src/main/java/nl/jiankai/mapper/converters/RefToObjectConverter.java

```diff
@@ -0,0 +1,32 @@
+package nl.jiankai.mapper.converters;
+
+import nl.jiankai.annotations.Converter;
+
+import java.sql.Ref;
+import java.sql.SQLException;
+
+@Converter(autoApply = true)
+public class RefToObjectConverter implements AttributeConverter<Ref, Object> {
+
+    @Override
+    public Object convert(Ref value) {
+        Object convertedValue = null;
+    try {
+        convertedValue = value.getObject();
```

Comment:
This will always map to a `Struct` object which is not very useful for the mapper since it's mapping to typed objects. So it will probably always fail